Contents/Code/PAutils.py

```diff
@@ -458,7 +458,7 @@ def cleanHTML(text):
 def getCleanSearchTitle(title):
     trashTitle = (
         'RARBG', 'COM', r'\d{3,4}x\d{3,4}', 'HEVC', r'H\d{3}', 'AVC', r'\dK',
-        r'\d{3,4}p', 'TOWN.AG_', 'XXX', 'MP4', 'KLEENEX', 'SD', 'HD',
+        r'\d{3,4}p', 'TOWN.AG_', 'MP4', 'KLEENEX', 'SD', 'HD',
```

Comment:
needed to match exact id e.g., https://realjamvr.com/scene/kill-bill-o-ren-ishii-a-xxx-parody/

Reply:
use then `searchData.filename` not `searchData.title`, it can break other sites

Reply:
reverted